tox.ini

```diff
@@ -14,7 +14,7 @@ commands=
     core: py.test {posargs:tests/core}
     pyethereum16: py.test {posargs:tests/backends/test_pyethereum16.py}
     pyethereum21: py.test {posargs:tests/backends/test_pyethereum21.py}
-    pyevm: py.test {posargs:tests/backends/test_pyevm.py}
+    pyevm: py.test {posargs:tests/backends/test_pyevm.py tests/backends/test_pyevm_serializer.py}
```

Comment:
If you put the test into `https://github.com/ethereum/eth-tester/blob/master/eth_tester/utils/backend_testing.py`, then it will get run by all backends, and this line change can be dropped.